### PR TITLE
Add SimulatorFM to Store

### PR DESCRIPTION
**Pull request by HuckinB**
This is the Fork for Simulator.FM
For more information contact me on Discord: HuckinB#0001

### CODE REVIEW

File: SimulatorFM/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "id": "228575716214702080"
+  },
+  "service": "SimulatorFM",
+  "description": {
+    "en": "Your Music, For Your Simulation."
+  },
+  "url": "simulator.fm",
+  "version": "1.0.1",
+  "logo": "https://panel.simulator.fm/public/avatars/default.jpg",
+  "thumbnail": "https://i.huckinb.com/t3xXxQc",
```
Comment (Bas950) [inline SimulatorFM/dist/metadata.json]: upload to imgur, or if huckinb is your own you can use that.

File: SimulatorFM/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "author": {
+    "name": "HuckinB",
+    "id": "228575716214702080"
+  },
+  "service": "SimulatorFM",
+  "description": {
+    "en": "Your Music, For Your Simulation."
+  },
+  "url": "simulator.fm",
+  "version": "1.0.1",
```
Comment (Bas950) [inline SimulatorFM/dist/metadata.json]: version should be 1.0 on first release

File: SimulatorFM/dist/metadata.json (hunk 1 above)
Comment (HuckinB) [reply SimulatorFM/dist/metadata.json]: huckinb.com is my domain yes

Comment (Bas950) [reply SimulatorFM/dist/metadata.json]: then upload the logo to it aswell

Comment (HuckinB) [reply SimulatorFM/dist/metadata.json]: For simplicity I uploaded them to both to Imgur

**Comment by HuckinB** [comment]
Proof of Working: https://i.imgur.com/kBxfJg1.png

**Comment by Bas950** [comment]
You need to upload it directly into the converstation... its a rule sorry xD

File: SimulatorFM/dist/metadata.json (hunk 1 above)
Comment (HuckinB) [reply SimulatorFM/dist/metadata.json]: Proof of Working: https://i.imgur.com/kBxfJg1.png

**Comment by Bas950** [comment]
xD like this:
Premid-logo-noback

**Comment by HuckinB** [comment]
Sorry my bad xD
Iv7EquSCc1

File: SimulatorFM/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/Zpdg2tx.png",
+  "thumbnail": "https://i.imgur.com/Iqy2rMp.png",
+  "color": "#65B6FF",
+  "tags": [
+    "SFM",
+    "ETS2",
+    "ATS",
+    "TMP",
+    "Your Music, For Your Simulation"
```
Comment (Alanexei) [inline SimulatorFM/dist/metadata.json]: Can you remove this? It's a little detailed for a tag. A one worder will work.

**Pull request merged**